android/tangram/jni/platform_android.cpp

```diff
@@ -72,6 +75,19 @@ void requestRender() {
     }
 }
 
+std::string systemFontPath(const std::string& _family, const std::string& _weight, const std::string& _style) {
+
+    jstring jfamily = jniEnv->NewStringUTF(_family.c_str());
+    jstring jweight = jniEnv->NewStringUTF(_weight.c_str());
+    jstring jstyle = jniEnv->NewStringUTF(_style.c_str());
+    jstring returnStr = (jstring) jniEnv->CallObjectMethod(tangramInstance, getFontFilePath, jfamily, jweight, jstyle);
+
+    size_t length = jniEnv->GetStringUTFLength(returnStr);
+    char* cdata = (char*) malloc(sizeof(char) * (length));
```

Comment:
This memory will be leaked after the function returns (`std::string` only copies the data, it doesn't take ownership). You could instead create a string of suitable length `auto out = std::string(length, 0)` and copy the data directly into it `jniEnv->GetStringUTFRegion(returnStr, 0, length, &out[0])`